azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -496,18 +498,14 @@ var disableFinishedJobs = function(data) {
       node.status = "READY";
       node.disabled = true;
     }
-    else if (node.status == "SUCCEEDED" || node.status=="RUNNING") {
+    else if (node.status == "SUCCEEDED" || node.missing) {
```

Comment:
Because if a job succeeded, we don't want to run that job again when preparing a new execution. Note that this function is only for the use case when you prepare a new execution based on a previous execution. Typically that only makes sense when some of the jobs were failed/cancelled/killed. If you prepare an execution based on a succeeded execution, then all jobs are initially disabled by this code.